src/webview-scripts/CallGraphRenderer.ts

```diff
@@ -137,7 +188,49 @@ export class CallGraphRenderer {
         this.render();
     }
 
+    timer: NodeJS.Timeout
+    visited: CallGraphLayoutNode[] = []
+
+    linkTarget = (hop: number, nodes: CallGraphLayoutNode[]) => {
+        this.visited.push(...nodes)
+        const kids: CallGraphLayoutNode[] = [];
+        this.linkSelection
+            .classed("link-source", (link) => {
+                const n = this.visited.find(n => n === link.source)
+                if ( n !== undefined) {
+                    if( this.visited.find(n => n === link.target) === undefined ) {
+                        kids.push(link.target)
+                        link.target.data.isTarget = true;
+                        link.target.data.hop = hop
+                    }
+                    return true
+                } else {
+                    return false
+                }
+            })
+            .classed("link-dimmed", (link) => {
+                return this.visited.find(n => n === link.source || n === link.target) === undefined
+            });
```

Comment:
I wonder: ESLint is a dev dep in this repo and there's a config file for it, yet there are so many wrongly formatted lines in your patch. This one, for example, is converted automatically on save for me:

```typescript
            .classed("link-dimmed", (link) => {
                return visited.find((n) => { return n === link.source || n === link.target; }) === undefined;
            })
```